ndn_python_repo/clients/getfile.py

```diff
@@ -38,30 +38,35 @@ async def fetch_file(self, name_at_repo: NonStrictName, local_filename: str = No
         :param local_filename: str. The filename of the retrieved file on the local file system.
         :param overwrite: If true, existing files are replaced.
         """
+
+        # If no local filename is provided, store file with last name component
+        # of repo filename
+        if local_filename is None:
+            local_filename = Name.to_str(name_at_repo)
+            local_filename = local_filename.strip().split("/")[-1]
```

Comment:
`os.path.basename()`?

Reply:
Gosh! Looking at the python API first makes life so much easier :D